controllers/nc-news-controller.js

```diff
@@ -27,10 +28,21 @@ exports.getArticleById = (req, res, next) => {
 };
 
 exports.getArticles = (req, res, next) => {
-    selectArticle().then((articles) => {
+  selectArticle()
+    .then((articles) => {
       res.status(200).send({ articles });
     })
     .catch((err) => {
       next(err);
+    });
+};
+
+exports.getComments = (req, res, next) => {
+  selectComments()
+    .then((comments) => {
+      res.status(200).send({ comments });
     })
+    .catch((err) => {
```

Comment:
We have a catch here, but no error handling tests. Make sure we are only writing code that is driven by testing.